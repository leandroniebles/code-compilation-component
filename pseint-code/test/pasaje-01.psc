subproceso lala(a por valor, b por referencia, c,d)
	a<-a+1;
	b<-b+2;
	c<-c+3;
	d[2]<-7;
FinSubProceso
Proceso sin_titulo
	xx<-10;
	yy<-20;
	zz<-30;
	Dimension ww[10];
	lala(xx,yy,zz,ww);
	escribir xx;
	escribir yy;
	escribir zz;
	escribir ww[2];
FinProceso
