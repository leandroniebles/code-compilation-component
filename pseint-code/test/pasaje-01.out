10
22
30
7
